### PR TITLE
Initial upload of HTML for GitHub Pages

### PR DESCRIPTION
**Pull request by ryanvergel**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,24 +1,9 @@
...
-ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
-LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE
-GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
-HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
-OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+The above copyright notice and this permission notice (including the next paragraph) shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (BiplabRaut) [inline LICENSE.txt]: This is the main license file that is for the entire library. 
I see that the BSD license is replaced with the MIT license, and it talks about both the software and documentation distribution.
Why is the main license modified ?
Was there any discussion on the license thing?
If it is needed only for the html and documentation, then should this license file be not placed in the doc folder ?

**Review by BiplabRaut — CHANGES_REQUESTED** [review]
I have a comment. Can you please respond.

**Comment by ryanvergel** [comment]
Hi @BiplabRaut , I have removed the superfluous `LICENSE.txt` file :)

**Comment by BiplabRaut** [comment]
> Hi @BiplabRaut , I have removed the superfluous `LICENSE.txt` file :)

Hi Ryan,
I see that you have now deleted the License file. But this file was the library level single master license file.
Are we not supposed to maintain the LICENSE info in the library level for the documentation?

**Review by BiplabRaut — APPROVED** [review]

**Pull request merged**